index.html

```diff
@@ -0,0 +1,145 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+
+  <body>
+    <div id="main">
```

Comment:
You have done an amazing job phrasing the statements like this. However, to prevent the student from feeling like they're being `ordered`, I'd encourage you  (and as per [guidelines](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/code-review/articles/give_code_review_basics.md)) 👨🏽‍💻  to start the request with phrases such as `please`, `kindly` ... e.g

> This is a polite suggestion: "I think it would be better to use semantic tags"
> This reads as an order: "Use semantic tags"

Otherwise, kudos on using the inline comments. You should use them again when making changes 👏🏽